src/poliastro/earth/__init__.py

```diff
@@ -0,0 +1,130 @@
+from functools import partial
+
+from astropy import units as u
+
+from poliastro.bodies import Earth
+from poliastro.core.perturbations import atmospheric_drag_model
+from poliastro.twobody.orbit import Orbit
+from poliastro.twobody.propagation import cowell
+
+
+class EarthSatellite:
+
+    """
+    Position and velocity of a body with respect to Earth
+    at a given time
+    """
+
+    def __init__(self, orbit):
+        """Constructor.
+
+        Parameters
+        ----------
+        orbit : Orbit
+            Position and velocity of a body with respect to an attractor
+            at a given time (epoch).
+
+        Raises
+        ------
+        TypeError
+            If the orbit is not an Orbit's instance.
+
+        """
+
+        if not isinstance(orbit, Orbit):
+            raise TypeError("Must receive an Orbit's instance")
+
+        self._orbit = orbit  # type: Orbit
+
+    @property
+    def orbit(self):
+        """Orbit of the Earth Satellite. """
+        return self._orbit
+
+    @staticmethod
+    @u.quantity_input(r=u.m, v=u.m / u.s)
+    def from_vectors(r, v):
+        """Return `Orbit` from position and velocity vectors.
+
+            Parameters
+            ----------
+            r : ~astropy.units.Quantity
+                   Position vector wrt attractor center.
+            v : ~astropy.units.Quantity
+                   Velocity vector.
+
+            Returns
+            -------
+            Orbit
+                New 'Orbit' position and velocity vectors.
+
+
+        """
```

Comment:
Misalignment of the docstring?

```suggestion
        """Return `Orbit` from position and velocity vectors.

        Parameters
        ----------
        r : ~astropy.units.
            Position vector wrt attractor center.
        v : ~astropy.units.Quantity
            Velocity vector.

        Returns
        -------
        Orbit
            New 'Orbit' position and velocity vectors.

        """
```